libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -514,16 +514,17 @@ public static string VerifyBoolean(object value, Expression expression)
         /// <param name="expression">Expression with children.</param>
         /// <param name="state">Global state.</param>
         /// <param name="verify">Optional function to verify each child's result.</param>
+        /// <param name="options">Options used in evaluation. </param>
         /// <returns>List of child values or error message.</returns>
-        public static (IReadOnlyList<object>, string error) EvaluateChildren(Expression expression, IMemory state, VerifyExpression verify = null)
+        public static (IReadOnlyList<object>, string error) EvaluateChildren(Expression expression, IMemory state, VerifyExpression verify, Options options)
```

Comment:
>VerifyExpression verify, Options options [](start = 115, length = 40)

Why not keep both optional?

Reply:
yeah, make sense, i thought it would be better to keep the order, so i remove the optional, we can move it to default. 

see Tom's comments about compatibility, do you have comments on that?